targets/ChibiOS/_Lwip/nf_lwipthread.c

```diff
@@ -546,6 +539,18 @@ static void do_reconfigure(void *p)
     chSemSignal(&reconf->completion);
 }
 
+static void initialize_sntp()
+{
+    sntp_stop();
+    sntp_setoperatingmode(SNTP_OPMODE_POLL);
+    sntp_servermode_dhcp(1); // try to get the ntp server from dhcp
```

Comment:
Per project code style: comment above line please
```suggestion
   // try to get the ntp server from dhcp
    sntp_servermode_dhcp(1);
```